src/vs/platform/opener/browser/link.ts

```diff
@@ -57,6 +59,11 @@ export class Link extends Disposable {
 			}
 		}));
 
+		const onMouseOver = domEvent(this.el, 'mouseover');
```

Comment:
I'd recommend to instead follow the existing pattern for styling links on hover using CSS, for example here's how the search view does it:
https://github.com/microsoft/vscode/blob/f547ada6eeb701a5e88cff1243d347f862d9d3c9/src/vs/workbench/contrib/search/browser/searchView.ts#L1977-L1981

Reply:
I implemented it but realized it doesn't really work because `Link` is used in a few different places:

- `src/vs/workbench/browser/parts/views`
- `src/vs/workbench/contrib/welcome/gettingStarted/browser`
- `src/vs/workbench/contrib/workspace/browser`

So it doesn't look like there's a safe CSS selector that can cover all those cases and potential future consumers of `Link`.

What's your thought?

Reply:
You might add a class to the link's `a` element and style based on that, but I'll leave further review to @joaomoreno as he owns this code.

Reply:
Oh yes, that sounds like the way to do it. I'll have a go while waiting for @joaomoreno's review

Reply:
No luck with `registerThemingParticipant` yet, I'll wait for @joaomoreno 's review for now

Reply:
Any thoughts @joaomoreno ?